### ISSUE TITLE
Gutenberg: hide "preview AMP" from the toolbar

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#42896]
With [AMP plugin 1.5.0](https://github.com/ampproject/amp-wp/releases/tag/1.5.0), we got "preview AMP" button now on the header toolbar on Atomic sites (which run the latest version of the plugin). We don't have that currently for simple sites, but it would pop up there too once we upgrade.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/87168/83545699-3bcf8a80-a508-11ea-9494-ebb6cf48db5c.png">
<img width="387" alt="image" src="https://user-images.githubusercontent.com/87168/83545789-602b6700-a508-11ea-8e5e-fb2c57811a4c.png">

(The icon to the left from "publish")

Clicking it opens a preview to show how post/page would look like on AMP.

It's a bit advanced feature for most .com customers and has too high prominence in the header toolbar. We'll need to find a better place for it.

@olaolusoga suggested moving to "Document" sidebar somehow. For now, we can just hide this and look at the placement together with other preview questions ahead (p58i-93P-p2).

**Comment by westonruter** [comment]
Yes, we agree. We actually tried to move it to the Preview dropdown but we weren't able to due to it not being extensible.

Here's the relevant PR for that preview button: https://github.com/ampproject/amp-wp/pull/4397

I inquired about moving to the Preview dropdown here: https://github.com/ampproject/amp-wp/issues/4368#issuecomment-597943645

If Gutenberg were updated to make this extensible we'd happily change the AMP plugin to integrate there.

**Comment by westonruter** [comment]
Aside: this preview AMP button only is present in Reader and Transitional template modes, since they entail paired AMP where there is a separate URL for AMP va the canonical non-AMP URL. In Standard mode, the AMP pages are canonical so there is no separate AMP preview button.

**Comment by westonruter** [comment]
Opened an issue for this: https://github.com/WordPress/gutenberg/issues/25309

**Comment by donlair** [comment]
@simison I'm going to remove this from the Flow Patrol board since the work in the PR immediately above here will enable the preferred long-term fix of adding AMP to the preview dropdown.
